common/script/public/userServices.js

```diff
@@ -29,6 +29,11 @@ angular.module('habitrpg')
       var settings = {}; //habit mobile settings (like auth etc.) to be stored here
       var user = {}; // this is stored as a reference accessible to all controllers, that way updates propagate
 
+      var userNotifications = {
+        "party.order" : "Party order updated.",
+        "party.orderAscending" : "Party order updated."
```

Comment:
Is it possible to make this message a translatable string? I know there's a few places in this file where we have fixed English strings, but we are likely to be fixing that over time, and it would be nice for new strings to start as translatable.